src/shared/default-routes.js

```diff
@@ -40,16 +41,16 @@ const DefaultRoutes = ({
       }
       <Route
         path='/'
-        component={FrontPage || Page}  />
+        component={fetchData(FrontPage || Page || Error)}  />
       <Route
         path=':page(/:subpage)'
-        component={Page} />
+        component={fetchData(Page || Error)} />
```

Comment:
Ah no, it's all good, we could probably do with a little more work in rendering the correct fallback component for production anyway, happy to mergeeeee this